README.md

```diff
@@ -46,6 +46,13 @@ It is distributed under the GNU GENERAL PUBLIC LICENSE Version 2 - please consul
     - ldap (users authentication)
     - openssl (encrypted communication)
 
+ * Supported browsers :
+    - IE 11+
+    - Edge
+    - Firefox (including ESR)
```

Comment:
For firefox, we could enforce that we support (the two) last ESR. At the moment, ESR45 is tested.
We have some css features who need recent version, i can't remember which version is needed but 45 is working.

Reply:
Updated :)